zeitgeschichte-wien.csl

```diff
@@ -0,0 +1,393 @@
+    <choose>
+      <if is-numeric="edition">
+        <number variable="edition" form="ordinal"/>
+        <text prefix=" " term="edition" form="short" suffix="."/>
+      </if>
+      <else-if variable="edition">
+        <text variable="edition"/>
+        <text prefix=" " term="edition" form="short" suffix="."/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="issued">
+    <choose>
+      <if type="book">
+        <text macro="edition" prefix=", "/>
+        <text prefix=" (" macro="collection" suffix=")"/>
+        <text prefix=", " variable="publisher-place"/>
+        <date prefix=" " variable="issued">
+          <date-part name="year"/>
+        </date>
+        <date prefix=" [" variable="original-date" suffix=")">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else-if type="chapter">
+        <text prefix=" (" macro="collection" suffix=")"/>
+        <text macro="edition" prefix=", "/>
+        <text prefix=", " variable="publisher-place"/>
+        <date prefix=" " variable="issued">
+          <date-part name="year"/>
+        </date>
+      </else-if>
+      <else-if type="article-journal article-magazine article-newspaper" match="any">
+        <group delimiter=" " prefix=", ">
+          <text variable="volume"/>
+          <group delimiter=" ">
+            <text variable="publisher-place"/>
+            <date variable="issued">
+              <date-part name="year" prefix=" (" suffix=")"/>
+            </date>
+          </group>
+          <text variable="issue"/>
+        </group>
+        <text macro="secondary-container-contributors" prefix="/"/>
+      </else-if>
+      <else>
+        <group delimiter=" " prefix=", ">
+          <text variable="publisher-place"/>
+          <choose>
+            <if variable="URL">
+              <text value="dort datiert"/>
+            </if>
+          </choose>
+          <date variable="issued" form="numeric">
+            <date-part name="day"/>
+            <date-part name="month"/>
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper chapter" match="any">
+        <text variable="page"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+        <group delimiter=" " prefix=" (" suffix=")">
+          <text term="accessed" text-case="lowercase"/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric" suffix="."/>
+            <date-part name="month" form="numeric" prefix=" " suffix="."/>
+            <date-part name="year" prefix=" "/>
+          </date>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <text variable="locator"/>
+  </macro>
+  <citation et-al-subsequent-min="6" et-al-subsequent-use-first="5" disambiguate-add-names="true">
```

Comment:
I don't see any mention of the et-al settings in the guide. You're sure about those?